spec/services/hesa_converter_spec.rb

```diff
@@ -0,0 +1,206 @@
+require 'rails_helper'
+
+RSpec.describe HesaConverter do
+  shared_examples 'convert sex field' do |data|
+    it "converts old hesa codes from #{data[:recruitment_cycle_year]} cycle for '#{data[:sex]}' into the most up to date HESA codes" do
+      recruitment_cycle_year = data[:recruitment_cycle_year]
+      application_form = create(:application_form, :completed)
+      application_form.update!(
```

Comment:
We can use `build` and `assign_attributes` here to speed up the tests significantly

Reply:
Good idea! Much faster 🚀